docs/conceptual/cmake-packages.rst

```diff
@@ -97,6 +97,10 @@ there's a catch-all, last resort variable consulted locating this file,
 ``-D CMAKE_HIP_COMPILER_ROCM_ROOT:PATH=`` which should be set the root of the
 ROCm installation.
 
+.. note::
+    Imported targets defined by `hip-lang-config.cmake` are for internal use for
```

Comment:
Can we just say: "Imported targets defined by `hip-lang-config.cmake` are for internal use only."

Reply:
Simplified.